<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LIVEcap</string>
  </property>
  <widget class="QWidget" name="operationWidget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>390</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>524</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="widthLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>宽度:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="widthSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="heightLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>高度:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="heightSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="frameLabel">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>帧率:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="frameSpinBox">
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Century Gothic&quot;;
background-color:#FFFFFF;  
border: 1px solid #AAB4C4;   
padding:0px;  
width: 75px;  
height:20px;  
border-radius:4px;  </string>
      </property>
      <property name="text">
       <string>开始录制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Century Gothic&quot;;
background-color:#FFFFFF;  
border: 1px solid #AAB4C4;   
padding:0px;  
width: 75px;  
height:20px;  
border-radius:4px;  </string>
      </property>
      <property name="text">
       <string>停止录制</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
